README.md

```diff
@@ -1,3 +1,121 @@
+
+2. Navigate to the project directory:
+`cd tester-tracker-sephora`
+
+3. Install the required gems:
+`bundle install`
+
+4. Set up the database:
+`rake sample_data`
+
+5. Start the Rails server:
+`bin/dev`
+
+## Usage
+
+1. Start the Rails server:
+`bin/dev or dev`
+
+2. Open browser and navigate to http://localhost:3000
+
+3. Use one of the following user profiles to interact with data-rich views or create an account and interact with the application using data seeded into your role's interface. 
+
+  NOTE: The credentials provided are for testing purposes only and do not correspond to any real Sephora accounts or information.
+
+- `username: 356177, password: password` : Operations interface
+- `username: 100000, password: password` : Admin interface
+- `username: 111111, password: password` : Lead interface
+- `username: 222222, password: password` : Beauty Advisor interface
+- `username: 333333, password: password` : Manager interface
+
+4. The search functionality assumes user has knowledge of Sephora's products(e.g. product names or SKUs). Refer to the `Makeup`, `Skincare`, `Hair`, `Fragrance` views to find examples of product names or SKUs available at user's store location. 
+
+5. Create a new tester in the `New` view. Use the products cards listed below the search bar to find a product you'd like to make a tester for.
+
+Example of the `New` view for creating testers:
+![alt text](tester-tracker-new-view-1.png)
+
+6. `Manage Testers` and `Trash Tester` views are available only for Sephora's Operations Associates and Admin. If you'd like to access these views, sign up as an Operations Associate or Admin. 
+
+7. As an Operations Associate, you can trash testers that have been placed onstage or aren't in condition to be an onstage tester.
+
+8. An Operations Associate can remove newly created testers that have been placed onstage from inventory by approving them.
+
+9. Admin and Operations teammembers can keep track of tester usage by visiting the `Trashed Testers` view.
+
+## Entity Relationship Diagram 
+
+Visual representation of Tester Tracker's database
+
+![alt text](erd-model.png)
+
+## Troubleshooting
+
+### Server not starting
+
+1. Ensure all required gems are installed: Run `bundle install` in terminal.
+
+2. Verify Ruby version (e.g. Ruby 3.x): Run `ruby -v` in terminal. Install correct version if needed.
+
+3. Restart the server `bin/server`.
+
+### Images or Assets not Loading
+
+1. Visit `app/assets/images` and verify that images are correctly saved in `images` or folder under `images`.
+
+2. In production.rb, ensure `config.assets.compile = true` is present.
+
+3. Restart the server `bin/server`.
+
+### Unable to log-in
+
+1. Run `rake sample_data` to populate database with seed data.
+
+2. Run `rails console` and query data (e.g. `Employee.all` or `Employee.count`) to ensure it was seeded correctly.
+
+## Contributing
+
+Contributions are welcome! Please follow these guidelines:
+
```

Comment:
make sure to include coding conventions, branch naming conventions, and the pull request process for contribution!